src/org/labkey/test/tests/issues/IssueAPITest.java

```diff
@@ -0,0 +1,280 @@
+package org.labkey.test.tests.issues;
+
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.issues.GetIssueCommand;
+import org.labkey.remoteapi.issues.GetIssueResponse;
+import org.labkey.remoteapi.issues.IssueCommand;
+import org.labkey.remoteapi.issues.IssueModel;
+import org.labkey.remoteapi.issues.IssueResponse;
+import org.labkey.remoteapi.issues.IssueResponseModel;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.InDevelopment;
+import org.labkey.test.pages.issues.DetailsPage;
+import org.labkey.test.util.APIUserHelper;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.IssuesHelper;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.labkey.test.util.PasswordUtil.getUsername;
+
+@Category({InDevelopment.class})
+public class IssueAPITest extends BaseWebDriverTest
+{
+    IssuesHelper _issuesHelper = new IssuesHelper(this);
+    APIUserHelper _userHelper = new APIUserHelper(this);
+    static String ISSUES = "issues";
+    static Integer TEST_USER_ID;
+    static String TEST_USER_NAME;
+    static String TEST_USER_DISPLAY_NAME;
+
```

Comment:
Why are these class variables static?

Reply:
so I didn't have to reference the instance while setting up the class, no other reason

Reply:
We make these sort of things static so that multiple test cases can share them. JUnit creates a new class instance for each test case.